modules/loader/src/main/java/com/opengamma/strata/loader/csv/FullSwapTradeCsvLoader.java

```diff
@@ -660,6 +663,23 @@ private static PriceIndexCalculationMethod parseInflationMethod(Optional<String>
         .map(cal -> BusinessDayAdjustment.of(dateCnv, cal));
   }
 
+  // days adjustment, defaulting business day convention and holiday calendar
+  private static BusinessDayAdjustment parseBusinessDayAdjustmentWithFallback(
```

Comment:
Probably should be `WithDefault` because "fallback" has a different meaning within this class.

Reply:
Removed this method completely.